### PR TITLE
Create Weekly Notification Message for Certbot Team

### PR DESCRIPTION
**Pull request by zoracon**
Composes Mattermost message to team channel.

- Scheduled job that is sent once a week with dynamically set date range of past 7 days.

### CODE REVIEW

File: .github/workflows/notify_weekly.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: Weekly Github Update
+
+on:
+  schedule:
+    # Every week on Thursday @ 1:00
+    - cron: "0 1 * * 4"
```
Comment (wgreenberg) [inline .github/workflows/notify_weekly.yaml:6]: do we want this at 1am? if we want 1pm, it should be `0 13 * * 4`

**Review by wgreenberg — CHANGES_REQUESTED** [review]
one comment + one question!

File: .github/workflows/notify_weekly.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+jobs:
+  send-mattermost-message:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Create Mattermost Message
+      run: |
+        DATE=$(date  --date="7 days ago" +"%Y-%m-%d")
+        MERGED_URL="https://github.com/pulls?q=merged%3A%3E${DATE}+org%3Acertbot+"
+        UPDATED_URL="https://github.com/pulls?q=updated%3A%3E${DATE}+org%3Acertbot+"
```
[the remark this answers is not in the document]
Comment (zoracon) [reply .github/workflows/notify_weekly.yaml:16]: Don't think so. I just copy & pasted the query link. I will adjust.

File: .github/workflows/notify_weekly.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (zoracon) [reply .github/workflows/notify_weekly.yaml:6]: I will adjust and also put `13:00` in the comment

Comment (zoracon) [inline .github/workflows/notify_weekly.yaml:6]:
```suggestion
    - cron: "0 13 * * 4"
```

File: .github/workflows/notify_weekly.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+name: Weekly Github Update
+
+on:
+  schedule:
+    # Every week on Thursday @ 1:00
```
Comment (zoracon) [inline .github/workflows/notify_weekly.yaml:5]:
```suggestion
    # Every week on Thursday @ 13:00
```

File: .github/workflows/notify_weekly.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+jobs:
+  send-mattermost-message:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Create Mattermost Message
+      run: |
+        DATE=$(date  --date="7 days ago" +"%Y-%m-%d")
+        MERGED_URL="https://github.com/pulls?q=merged%3A%3E${DATE}+org%3Acertbot+"
+        UPDATED_URL="https://github.com/pulls?q=updated%3A%3E${DATE}+org%3Acertbot+"
```
Comment (zoracon) [inline .github/workflows/notify_weekly.yaml:16]:
```suggestion
        MERGED_URL="https://github.com/pulls?q=merged%3A%3E${DATE}+org%3Acertbot"
        UPDATED_URL="https://github.com/pulls?q=updated%3A%3E${DATE}+org%3Acertbot"
```

**Review by wgreenberg — APPROVED** [review]

**Pull request merged**